tests/selenium/test_basic_copy_paste.py

```diff
@@ -0,0 +1,212 @@
+import dash
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+
+import dash_core_components as dcc
+import dash_html_components as html
+from dash_table import DataTable
+
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+
+import pandas as pd
+
+url = "https://github.com/plotly/datasets/raw/master/" "26k-consumer-complaints.csv"
+rawDf = pd.read_csv(url)
+df = rawDf.to_dict('rows')
+
+def get_app():
+    app = dash.Dash(__name__)
+
+    app.layout = html.Div(
+        [
+            DataTable(
+                id="table",
+                data=df[0:250],
+                columns=[{"name": i, "id": i, "hideable": i == "Complaint ID"} for i in rawDf.columns],
+                editable=True,
+                sort_action='native',
+                include_headers_on_copy_paste=True,
+            ),
+            DataTable(
+                id="table2",
+                data=df[0:10],
+                columns=[{"name": i, "id": i, "deletable": True} for i in rawDf.columns],
+                editable=True,
+                sort_action='native',
+                include_headers_on_copy_paste=True,
+            ),
+        ]
+    )
+
+    @app.callback(
+        Output("table", "data"),
+        [Input("table", "data_timestamp")],
+        [State("table", "data"), State("table", "data_previous")],
+    )
+    # pylint: disable=unused-argument
+    def update_data(timestamp, current, previous):
+        # pylint: enable=unused-argument
+        if timestamp is None or current is None or previous is None:
+            raise PreventUpdate
+
+        modified = False
+        if len(current) == len(previous):
+            for (i, datum) in enumerate(current):
+                previous_datum = previous[i]
+
+                if datum['Unnamed: 0'] != previous_datum['Unnamed: 0']:
+                    datum['Complaint ID'] = 'MODIFIED'
+                    modified = True
+
+        if modified:
+            return current
+        else:
+            raise PreventUpdate
+
+    return app
+
+
+def test_tbcp001_copy_paste_callback(test):
+    test.start_server(get_app())
+
+    with test.table('table') as target:
```

Comment:
I know we talked about this already but now that I see it used I'm not sure the context manager is really the best way to expose individual tables - (nearly?) all the tests just create that context up front and never do anything outside of it, so it feels like it's just adding a level of indentation for little reason. What if we just turn
```
with test.table('table') as source, test.table('table2') as target:
```
into
```
source, target = test.table('table'), test.table('table2')
```
?

Reply:
It is fairly idiosyncratic. I can 🔪 it

Reply:
https://github.com/plotly/dash-table/pull/696/commits/6c36f1d77f935d78004f595dada5b425d3d9652b